docs/IE_DG/OnnxImporterTutorial.md

```diff
@@ -2,7 +2,7 @@
 
 > **NOTE**: This tutorial is deprecated. Since OpenVINO™ 2020.4 version, Inference Engine enables reading ONNX models via the Inference Engine Core API
 > and there is no need to use directly the low-level ONNX* Importer API anymore. 
-> To read ONNX\* models, it's recommended to use the `Core::ReadNetwork()` method that provide a uniform way to read models from IR or ONNX format.
+> To read ONNX\* models, it's recommended to use the InferenceEngine::Core::ReadNetwork method that provide a uniform way to read models from IR or ONNX format.
```

Comment:
@ilya-lavrenov there needs to be a link to the API docs here. So user can figure out how to implement InferenceEngine::Core::ReadNetwork with ONNX model.

Reply:
@ryanloney Hi, user shouldn't implement `InferenceEngine::Core::ReadNetwork` it is already implemented. But if i remember right in our documentation we will have a link on API `InferenceEngine::Core::ReadNetwork` which will be generated by Doxygen.

Reply:
> our documentation we will have a link on API InferenceEngine::Core::ReadNetwork which will be generated by Doxygen.

Right. So, the link should be automatically generated